models/ClimateSensor.json

```diff
@@ -0,0 +1,86 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azsamples:iotdd:ClimateSensor;1",
```

Comment:
>iotdd [](start = 25, length = 5)

What does "iotdd" mean in this @id? Will customers know what it means?

Reply:
as I said, this is a placeholder until we agree on the id to use.